decidim-admin/app/cells/decidim/admin/results_per_page_cell.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class ResultsPerPageCell < Decidim::ViewModel
+      property :per_page, :per_page_range, :with_label
+
+      def path_for_num_per_page(num_per_page = per_page_range.first)
+        %(#{request.path}?per_page=#{num_per_page})
```

Comment:
For this to work with existing filters it should be changed to something like:

```ruby
controller.url_for(params.to_unsafe_h.merge(per_page: num_per_page))
```

Reply:
Well this was intended, as when the `:per_page` value changes the `:page` value has to be cleared. 
But yes maybe it could be removed only the `:page` and mantainthe rest of them.